### DISCUSSION TITLE
Can not enable Input Control on Pixel 5a 5G devices

### DISCUSSION THREAD

**Discussion by DroidShane** [rustdesk/rustdesk#5580] [General]
### Bug Description

Can not enable input control on  Pixel 5a 5G devices as you can not enable the Accessibility service for RustDesk. When you try to enable it you get the message "Restricted setting - For your security, this setting is currently unavailable." I tried this on more than one Pixel 5a 5G device with the same results. I have used Rust desk on Pixel 3a, Pixel 5, and Pixel 6 Pro devices with out this issue. Have only encountered it on the Pixel 5a 5G devices so far.

### How to Reproduce

Try to toggle on Input Control on a Pixel 5a 5G device. Follow the prompts to turn on Accessibility Service and discover it will not allow Accessibility Service.

### Expected Behavior

Expected Accessibility Service to be allowed.

### Operating system(s) on local side and remote side
Remote side Andriod 13, Local side Linux Ubuntu 22.04

### RustDesk Version(s) on local side and remote side

1.2.1 & 1.2.2

### Screenshots

![photo_2023-09-01_16-58-53](https://github.com/rustdesk/rustdesk/assets/143845158/d011ff95-2267-4afb-89dc-6b7748d470f7)
![photo_2023-09-01_16-58-58](https://github.com/rustdesk/rustdesk/assets/143845158/5ba4558e-bf5d-43c9-b487-c89b87d94bf7)
![photo_2023-09-01_16-59-03](https://github.com/rustdesk/rustdesk/assets/143845158/7c585ea8-2efc-4506-8ec1-dc326e2e6c1d)

**Comment by BrokenPen** [comment]
I dont have this phone, but I guess you may enable the input by 

https://stackoverflow.com/questions/72217216/android-13-enable-accessibility-service-for-apk-which-installed-not-from-googl

click on the app info, right top "three dot" -> "allow restricted settings"

**Comment by hatl** [comment]
I can confirm this issue on my Pixel 8 pro using rustdesk v1.2.3.
Other apps like bitwarden work fine - only rustdesk is affected.

@rustdesk could you please check?

**Comment by VoxBG** [comment]
> I dont have this phone, but I guess you may enable the input by
> 
> https://stackoverflow.com/questions/72217216/android-13-enable-accessibility-service-for-apk-which-installed-not-from-googl
> 
> click on the app info, right top "three dot" -> "allow restricted settings"

This worked for me, thank you!
